packages/components/src/notice/index.js

```diff
@@ -21,7 +21,7 @@ function Notice( { className, status, children, onRemove = noop, isDismissible =
 		'is-dismissible': isDismissible,
 	} );
 	return (
-		<div className={ classNames }>
+		<div className={ classNames } role="alert">
```

Comment:
Thank you for this improvement @afercia! I did some tests, and it looks like we are applying the role="alert" to a div that also contains a dismiss icon button. So for example in the max upload error message we get this as the alert message:
![image](https://user-images.githubusercontent.com/11271197/44819606-59f6b100-abe5-11e8-89ca-eee96f226dea.png)

Maybe we can apply the role to a div that does not contain the IconButton, so we avoid referencing the dismiss as part of the alert message.

(To easily replicate the problem we can limit the site upload limit by adding the following two lines to .htaccess file php_value upload_max_filesize 1M, php_value post_max_size 1M)

Reply:
@jorgefilipecosta thanks, yep noticed that and can replicate in an easier way (I'm on nginx - VVV, no .htaccess here 😉):

![screen shot 2018-08-30 at 08 57 19](https://user-images.githubusercontent.com/1682452/44834976-f6937000-ac32-11e8-8570-18035ed075fe.png)

The problem is if the notice content is _not_ a string, then there's no inner wrapper:
https://github.com/WordPress/gutenberg/blob/9a25ed1ae3f97e0888a62a53d09f484d31b9b7ad/packages/components/src/notice/index.js#L25

That leads me to the question: why `components-notice__content` shouldn't be rendered when the children is not a string?

Reply:
Ah hm seems to me that's because originally the wrapper was a `<p>` element, so it made sense to avoid to wrap other elements in a paragraph:
https://github.com/WordPress/gutenberg/commit/d64de0f04d3ff65473c8e53e92abb1cf92cc3d6f#diff-90222e8dfd0cf666132b45557dab1ba9R23

but now that the wrapper is a div, I think it can just be always rendered.